modules/TestUtils.js

```diff
@@ -28,6 +28,12 @@ export const isArray = (object) =>
   Array.isArray(object)
 
 /**
+ * Returns true if the given object is a set.
+ */
+export const isSet = (object) =>
+  Object.prototype.toString.call(object) === '[object Set]'
```

Comment:
There is only one possible solution, which I mentioned [earlier](https://github.com/mjackson/expect/pull/183#discussion_r95440134)